### ISSUE TITLE
GitHub deprecating support for actions using `node16`

### ISSUE THREAD

**Issue by caquino** [joelwmale/webhook-action#668]
Hi,

As part of some of my workflows, I use this action and I'm seeing warnings from GitHub about the deprecation of node16 support on actions.
```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: joelwmale/webhook-action@2.3.2. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
```

I am more than happy to submit a PR with the changes, but I see that the last update was 2 years ago so before doing the work I would like to ask if this is something that is welcome or if this action should be considered deprecated and I should look for a replacement/fork for this one.

Thanks in advance!

**Comment by joelwmale** [comment]
Hi @caquino. Always happy to take a PR to fix this up.

Project is not dead, just don't have any more features to add, so very little work needed to maintain.

Thanks!

**Comment by Gargaj** [comment]
Would it need more than updating https://github.com/joelwmale/webhook-action/blob/develop/action.yml#L24 from `node16` to `node20`?

**Comment by joelwmale** [comment]
@Gargaj I'll take a look at it in the next few hours and see if I can get it fixed for you.

**Comment by joelwmale** [comment]
@Gargaj this is all fixed and merged into master, at version 2.4.0.
